lib/msal-react-native-android-poc/README.md

```diff
@@ -0,0 +1,98 @@
+# MSAL React Native Proof of Concept (Android)
+MSAL for React Native is a proof of concept that allows React Native applications to authenticate users with Microsoft work and school accounts (AAD) as well as acquire access tokens from the Microsoft identity platform endpoint in order to access secured web APIs. Currently, this library is only available for the Android platform.
+## Getting started
+
+### Installation
+
+`$ npm install @azuread/msal-react-native-android-poc@1.0.3`
+
+### Register your application in the Azure Portal
+Under Authentication -> Platform configurations, click "Add a platform" and select Android. 
+To generate the base64 signature of your application:
+* MacOS: `keytool -exportcert -alias androiddebugkey -keystore ~/.android/debug.keystore | openssl sha1 -binary | openssl base64`
+* Windows: `keytool -exportcert -alias androiddebugkey -keystore %HOMEPATH%.android\debug.keystore | openssl sha1 -binary | openssl base64`
+
+For more information and help, refer to this [MSAL FAQ](https://github.com/AzureAD/microsoft-authentication-library-for-android/wiki/MSAL-FAQ).
+
+### Configuring in app
+Follow steps 1 through 3 in the [Using MSAL](https://github.com/AzureAD/microsoft-authentication-library-for-android#using-msal) section of MSAL for Android's README with these modifications/reminders:
+* In step 2, please name the config file `auth_config_single_account.json` and put it in `res/raw`. You can pretty much copy and paste the MSAL configuration generated for you in Azure Portal and refer to the [configuration file documentation](https://docs.microsoft.com/en-us/azure/active-directory/develop/msal-configuration).
+* In step 3, remember that for `android:path` you want to paste the base64 signature hash you generated in the console and not the url encoded version.
+
+## Usage
+Import MSAL for React Native as `MSAL`:
+```javascript
+import MSAL from '@azuread/msal-react-native-android-poc';
+```
+Importing will automatically create a `SingleAccountPublicClientApplication` with the configurations provided in `auth_config_single_account.json`.
+All of the MSAL methods return a promise. 
+You can use `MSAL` to sign in a user with [scopes](https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-permissions-and-consent#scopes-and-permissions) (a string of scopes separated by " "):
+```javascript
+try {
+    await MSAL.signIn(scopesValue);
+} catch (error) {
+    console.log(error);
+}
+```
+Most of the methods return a map, and in the case of `signIn`, a map containing the variables of an `IAccount` object is returned.
+We can access the username of the currently signed in user and log it to console (for example):
+```javascript
+try {
+    var account = await MSAL.signIn(scopesValue);
```

Comment:
I may just be ignorant of how react-native works under the hood, but is there a reason why all of these `var` can't be `const` or `let`?

Reply:
No good reason for `var`. I just read an article about when to use all three and concluded that pretty much all of them should be replaced with `const`. Thanks for pointing this out.